xbmc/Application.cpp

```diff
@@ -4085,16 +4085,24 @@ PlayBackRet CApplication::PlayFile(const CFileItem& item, bool bRestart)
       CJobManager::GetInstance().PauseJobs();
     }
 
-    // don't hold graphicscontext here since player
-    // may wait on another thread, that requires gfx
-    CSingleExit ex(g_graphicsContext);
+    if (item.Exists()) // this is also an implicit call to CWakeOnAccess::WakeUpHost() if enabled
+    {
+      // don't hold graphicscontext here since player
+      // may wait on another thread, that requires gfx
+      CSingleExit ex(g_graphicsContext);
 
-    iResult = m_pPlayer->OpenFile(item, options);
+      iResult = m_pPlayer->OpenFile(item, options);
+    }
+    else
+    {
+      CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Error, 
+        g_localizeStrings.Get(16026), g_localizeStrings.Get(16027));
+    }
   }
-  else
+
+  if (iResult != PLAYBACK_OK)
```

Comment:
You are right, I can revert that change to avoid the ErrorLog in that case. I am also not sure what is the best to do , pls let me know if I should da the change?